distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -276,13 +276,13 @@ type Number
 
              5.format "%x"
     format : Text -> Text
-    format fmt = String.format fmt this
+    format fmt = String.format fmt self
 
     ## Checks equality of numbers, using an `epsilon` value.
 
        Arguments:
        - that: The number to check equality against.
-       - epsilon: The value by which `this` and `that` can be separated by before
+       - epsilon: The value by which `self` and `that` can be separated by before
```

Comment:
I wonder what we should do about these?

`this` and `that` sounded nicely, now, not sure if we should change `that` to something else? But I'm not sure if there are good alternatives either.

Reply:
Yes, this is something @ekmett raised as well